### ISSUE TITLE
Realm crashes on statrup com.getkeepsafe.relinker.MissingLibraryException: lib/unknown/librealm-jni.so

### ISSUE THREAD

**Issue by DerPate** [realm/realm-java#3660]
Bug:
#### Goal
Start the app whiztout crashing the realm initprogress

#### Expected Results

> App starts

#### Actual Results

```java
E/AndroidRuntime: FATAL EXCEPTION: main
java.lang.RuntimeException: Unable to create application 
de.eplus.mappecc.client.android.B2PWhitelabelApplication: 
com.getkeepsafe.relinker.MissingLibraryException: lib/unknown/librealm-jni.so
     at android.app.ActivityThread.handleBindApplication(ActivityThread.java:3699)
     at android.app.ActivityThread.access$1200(ActivityThread.java:122)
     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1062)
     at android.os.Handler.dispatchMessage(Handler.java:99)
     at android.os.Looper.loop(Looper.java:132)
     at android.app.ActivityThread.main(ActivityThread.java:4123)
     at java.lang.reflect.Method.invokeNative(Native Method)
     at java.lang.reflect.Method.invoke(Method.java:491)
     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:841)
     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:599)
     at dalvik.system.NativeStart.main(Native Method)
  Caused by: com.getkeepsafe.relinker.MissingLibraryException: lib/unknown/librealm-jni.so
     at com.getkeepsafe.relinker.ApkLibraryInstaller.installLibrary(ApkLibraryInstaller.java:85)
     at com.getkeepsafe.relinker.ReLinkerInstance.loadLibraryInternal(ReLinkerInstance.java:180)
     at com.getkeepsafe.relinker.ReLinkerInstance.loadLibrary(ReLinkerInstance.java:136)
     at com.getkeepsafe.relinker.ReLinker.loadLibrary(ReLinker.java:70)
     at com.getkeepsafe.relinker.ReLinker.loadLibrary(ReLinker.java:51)
     at io.realm.internal.RealmCore.loadLibrary(RealmCore.java:59)
     at io.realm.Realm.init(Realm.java:188)
     at de.eplus.mappecc.client.android.B2PWhitelabelApplication.configureRealm(B2PWhitelabelApplication.java:74)
     at de.eplus.mappecc.client.android.B2PWhitelabelApplication.onCreate(B2PWhitelabelApplication.java:58)
     at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:968)
     at android.app.ActivityThread.handleBindApplication(ActivityThread.java:3696)
     at android.app.ActivityThread.access$1200(ActivityThread.java:122) 
     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1062) 
     at android.os.Handler.dispatchMessage(Handler.java:99) 
     at android.os.Looper.loop(Looper.java:132) 
     at android.app.ActivityThread.main(ActivityThread.java:4123) 
     at java.lang.reflect.Method.invokeNative(Native Method) 
     at java.lang.reflect.Method.invoke(Method.java:491) 
     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:841) 
     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:599) 
     at dalvik.system.NativeStart.main(Native Method)

```
#### Steps & Code to Reproduce

Start an Android App in an Project with an lower API level then 14

#### Version of Realm and tooling
Realm version(s): 2.0.1

Android Studio version: 2.2.1

Which Android version and device: Multiple API level below 14

**Comment by Zhuinden** [comment]
What device is this? It doesn't seem to find its architecture.

**Comment by DerPate** [comment]
this was on an emulator to reproduce it

**Comment by Henkoglobin** [comment]
It was an emulator with API level 13

**Comment by zaki50** [comment]
API Level 13 of the emulator only supports `armeabi`, but Realm Java 2.0.0 or later does not provide `armeabi` binary. ( https://github.com/realm/realm-java/blob/v2.0.0/CHANGELOG.md#breaking-changes )
That's the reason why the emulator can't load the native library.
Currently running on the API 13 emulator is not supported.

We are discussing about getting back `armeabi` on #3506

**Comment by Bertkiing** [comment]
I also meet the stupid problem....

**Comment by pranitsawantAtBwellthy** [comment]
is this solved ??

**Comment by Zhuinden** [comment]
Shouldn't happen above api level 14.

**Comment by softwarejoint** [comment]
Device; TrendMicro 
Model: GI-I9500_TMMARS
Android: 5.1

Got this crash

Caused by com.getkeepsafe.relinker.MissingLibraryException: librealm-jni.so
       at com.getkeepsafe.relinker.ApkLibraryInstaller.installLibrary(SourceFile:123)
       at com.getkeepsafe.relinker.ReLinkerInstance.loadLibraryInternal(SourceFile:180)
       at com.getkeepsafe.relinker.ReLinkerInstance.loadLibrary(SourceFile:136)
       at com.getkeepsafe.relinker.ReLinker.loadLibrary(SourceFile:70)
       at com.getkeepsafe.relinker.ReLinker.loadLibrary(SourceFile:57)
       at io.realm.internal.RealmCore.loadLibrary(SourceFile:60)
       at io.realm.Realm.init(SourceFile:269)

**Comment by patelnishantk** [comment]
Our Fabric shows same stack trace as show by @softwarejoint. 

GI-I9500_TMMARS is the only device that faces this crash. We use Realm v5.9.0 that uses ReLinker v1.3.0. As per ReLinker, the fix for split APK is part of 1.3.0 so don't know why it still crashes on this device.

One thing that I observed in fabric is the device is rooted.
